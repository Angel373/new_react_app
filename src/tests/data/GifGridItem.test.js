import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import React from 'react';
import {GifGridItem} from '../../components/GifGridItem';

describe('Pruebas en <GifGridItem />', () => {
    const title = 'holamundo';
    const url = 'https://localhost.cualquier.com'
    const wrapper = shallow( <GifGridItem title={title} url={url}/>);

    test('debe de mostrar el componente <GifGridItem /> correctamente', () => {

        expect(wrapper).toMatchSnapshot();
    })

    test('debe de mostrar la clase del div animate__fadeIn', ()=>{
        const findClass = 'animate__fadeIn';
        const div = wrapper.find('div');
        const hasClass = div.prop('className');


        expect(hasClass.includes(findClass)).toBe(true);
    })
    
})
