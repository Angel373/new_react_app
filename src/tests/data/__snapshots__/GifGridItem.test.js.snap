// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> debe de mostrar el componente <GifGridItem /> correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="holamundo"
    src="https://localhost.cualquier.com"
  />
  <p>
     
    holamundo
     
  </p>
</div>
`;
